Assets/Scripts/Models/Area/World.cs

```diff
@@ -478,6 +484,7 @@ private void TickEveryFrame(float deltaTime)
     {
         CharacterManager.Update(deltaTime);
         FurnitureManager.TickEveryFrame(deltaTime);
+        GameEventManager.Update(deltaTime);
         ShipManager.Update(deltaTime);
```

Comment:
Do we need to update the GameEvents every frame? As far as I know they're not the sort of thing that needs that degree of granularity, they're just supposed to either just do it (in the case of spawning crew) or start the events going (such as maybe a pirate attack or a trader visiting)